bin/sqlline.py

```diff
@@ -114,6 +116,7 @@ def kill_child():
     disable_jna = ""
 
 java_cmd = java + ' $PHOENIX_OPTS ' + \
+    (args.debug and "-Xrunjdwp:transport=dt_socket,server=y,suspend=n,address=127.0.0.1:8071" or "") + \
```

Comment:
this is local change right?

Reply:
I am hoping to include this as it is quite useful. Do you see any problem?